plugins/woocommerce/src/StoreApi/Routes/V1/Agentic/CheckoutSessions.php

```
@@ -0,0 +1,425 @@
+	}
+
+	/**
+	 * Check if the request is authorized.
+	 *
+	 * V1 implementation: Return true for now (skip auth check).
+	 * Future: Implement Bearer token authentication.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 * @return bool True if authorized.
+	 */
+	public function is_authorized( \WP_REST_Request $request ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)